source/kit/components/ui/components.ui.config.js

```diff
@@ -21,4 +21,25 @@ angular.module('msm.components.ui')
 .config(function(uibDatepickerPopupConfig) {
   uibDatepickerPopupConfig.showButtonBar = false;
   uibDatepickerPopupConfig.datepickerPopupTemplateUrl = 'components/ui/msm-datepicker/msm-datepicker-popup.html';
+})
+
+.constant('uiDatetimePickerConfig', {
```

Comment:
Yes, it's just the default settings. Customization is possible, e.g. directly on the element:
`<input datetime-picker="..."  enable-time="false"/>`